cmd/analyseError_test.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"bytes"
+	"errors"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/damianoneill/nc-hammer/result"
+	"github.com/stretchr/testify/assert"
+)
+
+var myCmd = analyseErrorCmd
+
+func Test_Args(t *testing.T) {
+	t.Run("len(arg) != 1", func(t *testing.T) {
+		args := []string{}
+		errLen := strconv.Itoa(len(args))
+		analyseErrorCmd.Args(myCmd, args)
+
+		assert.Equal(t, analyseErrorCmd.Args(myCmd, args), errors.New("error command requires a test results directory as an argument"), "failed"+errLen)
+	})
+	t.Run("len(arg) == 1", func(t *testing.T) {
+		args := []string{"arg1"}
+		errLen := strconv.Itoa(len(args))
+		analyseErrorCmd.Args(myCmd, args)
+		assert.Equal(t, analyseErrorCmd.Args(myCmd, args), nil, "failed"+errLen)
+	})
+}
+
+func Test_Run(t *testing.T) {
```

Comment:
Since your not commenting the test, think better how you could name the function to convey what your trying to test.